src/styles/components/Select.scss

```diff
@@ -0,0 +1,77 @@
+@import "../common/fonts";
+@import "../common/layout";
+@import "../common/colors";
+
+
+.clr-form-control {
+
+    .clr-control-label {
```

Comment:
same class is present in other SCSS files as well. Check if we can create mixin for it.

Reply:
Also please make sure other component styles not getting broken as its not specific for selector for 'select'

Reply:
done

Reply:
there is a styling issue i saw at the input and the button in create object store..but i dont think that is related to any styling included for select.
![image](https://user-images.githubusercontent.com/95674930/149307622-76a295ef-d690-4f4b-bb74-8a13db4744ac.png)